capstone-app/src/app/modules/maps/map/map.component.scss

```diff
@@ -1,36 +1,177 @@
-.map-container {
-  margin-left: 43px;
-  margin-right: 43px;
+.search-icon {
+  margin-right: -30px;
 }
 
-.search-bar {
-  padding-bottom: 0;
-  text-align: center;
+.flex-menu {
+  align-content: center;
+  align-items: center;
+  display: flex;
 }
 
-.search-icon {
-  align-self: flex-start;
+mat-icon {
+  margin-right: 20px;
+  vertical-align: middle;
+}
+
+mat-icon:hover {
+  color:#1A73E9;
+}
+
+mat-form-field:focus-within .search-icon {
+  color:#1A73E9;
+}
+
+mat-icon .search-box {
+  margin-right: -20px;
 }
 
 .search-box {
-  display: flex;
-  flex-wrap: nowrap;
+  font-size: 15px;
+  height: 40px !important;
+  margin-bottom: -5px;
```

Comment:
Avoid negative margin

Reply:
For these negative margins, they were necessary because there was a lot of angular material card built in formatting I had to try to overrirde. Im strugging to come up with an alternative here.